rapids-cmake/cuda/set_architectures.cmake

```diff
@@ -56,26 +56,32 @@ Result Variables
 function(rapids_cuda_set_architectures mode)
   list(APPEND CMAKE_MESSAGE_CONTEXT "rapids.cuda.set_architectures")
 
-  set(supported_archs "70" "75" "80" "86" "90" "100" "120")
-
-  if(CMAKE_CUDA_COMPILER_ID STREQUAL "NVIDIA" AND CMAKE_CUDA_COMPILER_VERSION VERSION_LESS 12.0.0)
-    rapids_cmake_policy(DEPRECATED_IN 25.08 REMOVED_IN 25.12
-                        MESSAGE "Support for CUDA versions below 12 has been deprecated")
-  endif()
-  if(CMAKE_CUDA_COMPILER_ID STREQUAL "NVIDIA" AND CMAKE_CUDA_COMPILER_VERSION VERSION_LESS 11.1.0)
-    list(REMOVE_ITEM supported_archs "86")
-  endif()
-  if(CMAKE_CUDA_COMPILER_ID STREQUAL "NVIDIA" AND CMAKE_CUDA_COMPILER_VERSION VERSION_LESS 11.8.0)
-    list(REMOVE_ITEM supported_archs "90")
-  endif()
-  if(CMAKE_CUDA_COMPILER_ID STREQUAL "NVIDIA" AND CMAKE_CUDA_COMPILER_VERSION VERSION_LESS 12.8.0)
-    list(REMOVE_ITEM supported_archs "120")
-    list(REMOVE_ITEM supported_archs "100")
-  endif()
-
-  if(CMAKE_CUDA_COMPILER_ID STREQUAL "NVIDIA" AND CMAKE_CUDA_COMPILER_VERSION VERSION_GREATER_EQUAL
-                                                  13.0.0)
-    list(REMOVE_ITEM supported_archs "70")
+  if(CMAKE_CUDA_COMPILER_ID STREQUAL "NVIDIA")
+
+    if(CMAKE_CUDA_COMPILER_VERSION VERSION_GREATER_EQUAL 13.0.0)
+      set(supported_archs "75" "80" "86" "90a" "100f" "120a" "121f")
```

Comment:
Does this give us 121 PTX that would be forward-compatible beyond the 120 family? I think we still need that for CUDA >=13.0.0 and ==12.9.0.

Reply:
I see a couple of other issues as well with < 12.8. Working on another round of changes